### ISSUE TITLE
Close inspector when the connected tab is closed

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#5097]
(This should be looked at after #4996 is in)

Currently, if a tab is closed and the inspector is open, it stays open until it's at some point destroyed by Qt/Python.

This also seems to lead to weird behavior in some scenarios when quitting/relaunching - from IRC:

```
18:11 <cetex> hm. accidentally closed window while i had webinspector open, then tried launching qutebrowser again, realized my mistake, closed webinspector and then relaunched qutebrowser and all my tabs are gone..
[...]
18:19 <cetex> auto_save.session is set, had webinspector open, closed qutebrowser window, tried relaunching qutebrowser, nothing happened, closed webinspector, reopened qutebrowser -> everything gone.
18:20 <The-Compiler> cetex: hmm, I now tried with the re-launching, and then qutebrowser actually quits here (i.e. the inspector closes), weird
```

**Comment by The-Compiler** [comment]
Duplicate of #4348
